src/_posts/platform/getting-started/2000-01-01-how-to-migrate-from-heroku.md

```diff
@@ -5,43 +5,78 @@ tags: heroku tutorial
 index: 11
 ---
 
-Scalingo is a Platform as a Service [highly compatible with Heroku]({% post_url platform/getting-started/2000-01-01-heroku-compatibility %}). It's very easy to understand and you shouldn't have any problem migrating your app from Heroku to Scalingo.
+Scalingo is a Platform as a Service [highly compatible with Heroku]({% post_url platform/getting-started/2000-01-01-heroku-compatibility %}). This page is here to help your migrate from Heroku to Scalingo as easir as possible.
 
-It is so easy that you can use [heroku2scalingo](https://github.com/Scalingo/heroku2scalingo/releases/latest), a tool to migrate a Heroku app to Scalingo.
+## Requirements
 
-## heroku2scalingo
+To be able to follow this tutorial, we assume that you have:
 
-### Usage
+- A Scalingo account
+- The Scalingo [CLI]({% post_url platform/cli/2000-01-01-start %}) installed
+- A local copy of the repository containing your app’s code
+
+Please ensure you are running all commands above from the root of the repository containing your app’s code. 
 
 ```bash
-./heroku2scalingo <herokuAppName>
+$ cd <app directory>
 ```
```

Comment:
Not sure this code block is really useful...

Reply:
I wondered the same, but I like that it state "every" command. Can be useful for beginners.